scripts/lint_site.sh

```diff
@@ -18,10 +18,18 @@ set -e
 
 FAILED=0
 
+BRANCH="${BRANCH:-master}"
+
 if [[ "$#" -ne 0 ]]; then
     LANGS="$*"
 else
-    LANGS="en zh"
+    files_changed="$(git diff --name-only --diff-filter=b "$(git merge-base HEAD "$BRANCH")")"
+    if (echo "${files_changed}" | grep -q /en/); then
+        LANGS+="en "
+    fi
+    if (echo "${files_changed}" | grep -q /zh/); then
+        LANGS+="zh"
+    fi
 fi
```

Comment:
This is very cool, might as well add pt-br here.
```suggestion
    if (echo "${files_changed}" | grep -q /pt-br/); then
        LANGS+="pt-br"
    fi
fi
```

Reply:
Pushed commit separately since this was actually missing two spaces.